included/12.2.0/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 12.2.0 cypress/browsers:node16.18.0-chrome107-ff106-edge
+#
+# build this image with command
+#   docker build -t cypress/included:12.2.0 .
+#
+FROM cypress/browsers:node16.18.0-chrome107-ff106-edge
```

Comment:
There was an ask in `#support` yesterday regarding Chrome 108 images. Worth updating?

Reply:
Probably outside the scope of this PR, just curious how/when the decision to bump is made

Reply:
I think that @mjhenkes is working on some stuff with regards to this? Not sure if we should update now or wait, any thoughts from other people more familiar with the process/issues?

Reply:
I'd update it in a different PR, not tied to the release. and yes I am working on a new way to generate docker images.

Reply:
I'm actually not sure with how this repo is setup if we even could update two docker images in the same PR